extensions/amp-access/0.1/amp-access.js

```diff
@@ -144,20 +156,54 @@ export class AccessService {
    * @private
    */
   buildConfig_() {
-    let config;
+    let configJson;
     try {
-      config = JSON.parse(this.accessElement_.textContent);
+      configJson = JSON.parse(this.accessElement_.textContent);
     } catch (e) {
       throw new Error('Failed to parse "amp-access" JSON: ' + e);
     }
-    return {
-      authorization: assertHttpsUrl(assert(config['authorization'],
-          '"authorization" URL must be specified')),
-      pingback: assertHttpsUrl(assert(config['pingback'],
-          '"pingback" URL must be specified')),
-      login: assertHttpsUrl(assert(config['login'],
-          '"login" URL must be specified')),
+
+    // Parse type.
+    let type = null;
+    const typeStr = configJson['type'];
+    if (typeStr) {
+      for (const k in AccessType) {
```

Comment:
This loops seems unnecessary for now, since we could just use `typeStr.toLowerCase()`.